tyr/servers/server.py

```diff
@@ -199,6 +200,41 @@ def configure(self):
         self.log.info('Using Chef path "{path}"'.format(
                                 path = self.chef_path))
 
+        if self.dns_zones is None:
+            self.log.warn('No DNS zones specified')
+            self.dns_zones = [
+                {
+                    'id': {
```

Comment:
It wouldn't be hard to move into a configuration file, but I'd like to spend some time thinking about how the file would be formatted and structured before I start moving stuff into it.